README.md

```diff
@@ -0,0 +1,87 @@
+
+# Slurm charms
+
+[Juju](https://juju.is) charms for automating the Day 0 to Day 2 operations of the [Slurm workload manager](https://slurm.schedmd.com/overview.html) ⚖️🐧
+
+[![CI](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
+
+The `slurm-charms` repository provides many of the Slurm daemons that are required to deploy, configure and manage a Slurm-based HPC cluster:
+
+- [`slurmctld`](./charms/slurmctld/): the central management daemon for Slurm.
+- [`slurmd`](./charms/slurmd): the compute node daemon for Slurm.
+- [`slurmdbd`](./charms/slurmdbd): the database daemon for Slurm.
+- [`slurmrestd`](./charms/slurmrestd/): the REST API interface to Slurm.
+
+## ✨ Getting started
+
+This should be run using Juju 3.x or greater.
+
+```shell
+$ juju deploy slurmctld --channel edge
+$ juju deploy slurmd --channel edge
+$ juju deploy slurmdbd --channel edge
+$ juju deploy slurmrestd --channel edge
+$ juju deploy mysql --channel 8.0/stable
+$ juju deploy mysql-router slurmdbd-mysql-router --channel dpe/edge
+
+$ juju integrate slurmctld:slurmd slurmd:slurmctld
+$ juju integrate slurmctld:slurmdbd slurmdbd:slurmctld
+$ juju integrate slurmctld:slurmrestd slurmrestd:slurmctld
+$ juju integrate slurmdbd-mysql-router:backend-database mysql:database
+$ juju integrate slurmdbd:database slurmdbd-mysql-router:database
```

Comment:
Since we don't have the shell prompt for the `tox` commands, should we remove them here as well?

```suggestion
juju deploy slurmctld --channel edge
juju deploy slurmd --channel edge
juju deploy slurmdbd --channel edge
juju deploy slurmrestd --channel edge
juju deploy mysql --channel 8.0/stable
juju deploy mysql-router slurmdbd-mysql-router --channel dpe/edge

juju integrate slurmctld:slurmd slurmd:slurmctld
juju integrate slurmctld:slurmdbd slurmdbd:slurmctld
juju integrate slurmctld:slurmrestd slurmrestd:slurmctld
juju integrate slurmdbd-mysql-router:backend-database mysql:database
juju integrate slurmdbd:database slurmdbd-mysql-router:database
```

Ik that `markdownlint` - which we're not using - will complain if you have the shell prompt `$` but don't have any example output shown.